.env

```diff
@@ -18,6 +18,8 @@ SCALA_BINARY_VERSION=2.12
 SPARK_VERSION=3.4.2
 SPARK_BINARY_VERSION=3.4
 SPARK_HADOOP_VERSION=3.3.4
+FLINK_BINARY_VERSION=1.18
+FLINK_VERSION=1.18.0
```

Comment:
1.18.1 is available https://dlcdn.apache.org/flink/flink-1.18.1/

Reply:
supported 1.18.1